.eslintrc.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Please keep this file in js format. This is the standard format for other projects in Amsterdam.